### ISSUE TITLE
Remove legacy disclosure widgets

### ISSUE THREAD

**Issue by joedolson** [joedolson/my-calendar#683]
**Is your feature request related to a problem? Please describe.**
The legacy disclosure widgets are the less accessible option, only maintained for backwards compatibility. They add a significant amount of maintenance overhead, due to the differences in output and implementation in both PHP and JS. Modals have been the default now since 3.5.0. 

In advance: examine the design differences between the two, and decide whether they merit any design options. Can auto toggle based on the existing values.

**Comment by joedolson** [comment]
Practically speaking, I should leave the list view disclosure widget as an option, and remove the grid view option. The list view serves a different purpose, and doesn't have the accessibility and usability problems that the grid widget has.
